README.md

```diff
@@ -8,11 +8,25 @@ In order to be able to use Skrop, you have to be familiar with how
 [Skipper](https://github.com/zalando/skipper) works.
 
 ### Install dependencies:
+
+#### System dependencies
+
+The [vips](https://github.com/jcupitt/libvips) library needs to be installed for Skrop to build.
+
+On macOS, that can be easily done using `brew`.
```

Comment:
can you please add the info for linux? it's enough to run packaging/install-vips.sh

Reply:
With pleasure!

Reply:
@rgritti : Done.